src/views/account/Homeworks/Homeworks.tsx

```diff
@@ -87,18 +87,26 @@ const WeekView: Screen<"Homeworks"> = ({ route, navigation }) => {
   }
   const firstDateEpoch = dateToEpochWeekNumber(firstDate);
 
-  // Function to get the current week number since epoch
-  const getCurrentWeekNumber = () => {
+  const getWeekNumber = (homeworks) => {
```

Comment:
```suggestion
  const getWeekNumber = (homeworks: Record<number, Homework[]>) => {
```
par rapport à TypeScript

Reply:
oue